stubs/dasbus/connection.pyi

```diff
@@ -0,0 +1,48 @@
+from typing import Callable, Dict, List, Optional
```

Comment:
stubs/ just contains the minimal stubs needed to satisfy mypy and eliminate all dynamic typing. There are some cheats and such in here, but that shouldn't be an issue.